src/libraries/Microsoft.PowerFx.Core/Binding/BinderUtils.cs

```diff
@@ -140,7 +140,19 @@ internal static bool TryGetBestOverload(TexlBinding txb, CallNode node, DType[]
                 var localWarnings = new LimitedSeverityErrorContainer(txb.ErrorContainer, DocumentErrorSeverity.Warning);
 
                 // Typecheck the invocation and infer the return type.
-                typeCheckSucceeded = maybeFunc.HandleCheckInvocation(txb, args, argTypes, localWarnings, out returnType, out nodeToCoercedTypeMap);
+                //typeCheckSucceeded = maybeFunc.HandleCheckInvocation(txb, args, argTypes, localWarnings, out returnType, out nodeToCoercedTypeMap);
+
+                typeCheckSucceeded = HandleCheckInvocationWithDeferred(maybeFunc, txb, args, argTypes, out var checkInvocationErrors, out returnType, out nodeToCoercedTypeMap);
```

Comment:
>HandleCheckInvocationWithDeferred [](http://example.com/codeflow?start=37&length=33)

Tracking for future improvement once dataflow is ready: 
[https://github.com/microsoft/Power-Fx/issues/837](https://github.com/microsoft/Power-Fx/issues/837)